autogen/main/outputs.tf.tmpl

```diff
@@ -114,10 +114,15 @@ output "node_pools_names" {
 }
 
 output "node_pools_versions" {
-  description = "List of node pools versions"
+  description = "Node pool versions by node pool name"
```

Comment:
We'll want to note this is now a map in the upgrade notes.

Reply:
This should already be done (in the new `upgrading_to_v18.0.md` file).

Reply:
Ah I missed you'd started the upgrade guide. Thank you for working on that!